services/web/src/helpers/modal.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Modal } from 'semantic';
+
+export default function modal(Component) {
+  class ModalWrapper extends React.Component {
```

Comment:
could we call this `WrappedModal` to really drive home that the outer wrapper IS the modal? We could `Component` `ViewComponent` instead... If someone comes in here wondering what's going on I just want to really make the intent clear.